### ISSUE TITLE
Add class to .ce_player if it has video or audio

### ISSUE THREAD

**Issue by Aybee** [contao/contao#441]
**Description**  
Wäre es möglich dem `div.ce_player` entsprechend des Inhalts die Klassen `video` bzw. `audio` zu vergeben?

**Example** 
Der Wrapper für ein Video wird meist responsive gestaltet und benötigt daher z.B. so ein CSS
```css
.ce_player.video {
  position: relative;
  padding-bottom: 56.25%;
}
.ce_player video {
  position: absolute;
  top: 0;
  bottom: 0;
  margin: auto;
  max-width: 100%;
  height: auto;
}
```
Momentan suche ich nach `video`s innerhalb von `.ce_player` und gebe dem `parent `die Klasse `video`
```js
$('video').parent('.ce_player').addClass('video');
$('audio').parent('.ce_player').addClass('audio');
```

**Comment by Aybee** [comment]
Thank you. Maybe the easiest way without BC break would be to add a DIV wrapper into the template.

**Comment by leofeyer** [comment]
I just realized that this can already be accomplished by adding the CSS class in the back end settings:

<img width="609" alt="" src="https://user-images.githubusercontent.com/1192057/59922281-b9ea6c80-9430-11e9-8bf2-1531c82e9641.png">

**Comment by asaage** [comment]
I think here was asked for an automated css-class.
Css-classnames which you put into the expert-settings don't have any relation to the media/mime-type.
Maybe using a definite class at the figure-element would make sense.
https://github.com/contao/contao/blob/master/core-bundle/src/Resources/contao/templates/elements/ce_player.html5#L5

```diff
--- <figure class="video_container">
+++ <figure class="<?php if ($this->isVideo): ?>video<?php else: ?>audio<?php endif; ?>_container">
```

**Comment by fritzmg** [comment]
Yeah, it's wrong that the player currently always uses a `video_container` class, since it can be an audio player as well.

**Comment by leofeyer** [comment]
Implemented in 6b12366f143a87318ca761e004c31b815e9dc0cf.

**Comment by Aybee** [comment]
Thank you very much.
